File: frontend-android/app/src/main/java/com/viewpharm/yakal/type/ETakingTime.kt
package com.viewpharm.yakal.type

enum class ETakingTime(private val time: String) {
    MORNING("아침"),
    AFTERNOON("점심"),
    EVENING("저녁"),
    DEFAULT("기타"),
    INVISIBLE("빈칸");

    override fun toString(): String {
        return time
    }
}

File: frontend-hybrid/yakal/lib/models/Home/e_taking_time.dart
enum ETakingTime {
  MORNING,
  AFTERNOON,
  EVENING,
  DEFAULT,
  INVISIBLE;
}

extension ETakingTimeExtension on ETakingTime {
  String get time {
    switch (this) {
      case ETakingTime.MORNING:
        return "아침";
      case ETakingTime.AFTERNOON:
        return "점심";
      case ETakingTime.EVENING:
        return "저녁";
      case ETakingTime.DEFAULT:
        return "기타";
      case ETakingTime.INVISIBLE:
        return "빈칸";
      default:
        return "";
    }
  }
}
